python/cuml/cuml/svm/linear_svr.py

```diff
@@ -138,6 +140,33 @@ def __init__(self, **kwargs):
 
         super().__init__(**kwargs)
 
+    @classmethod
+    def _params_from_cpu(cls, model):
+        return {
+            "loss": model.loss,
+            "epsilon": model.epsilon,
+            **super()._params_from_cpu(model),
+        }
+
+    def _params_to_cpu(self):
+        return {
+            "loss": self.loss,
+            "epsilon": self.epsilon,
+            "dual": False,  # cuML LinearSVR is primal
+            **super()._params_to_cpu(),
+        }
+
+    def _attrs_from_cpu(self, model):
+        if hasattr(model, "dual") and model.dual:
+            raise UnsupportedOnGPU(
+                "sklearn LinearSVR has 'dual' (default True) but cuML LinearSVR is primal"
+            )
+
+        return super()._attrs_from_cpu(model)
```

Comment:
We shouldn't need to fallback on the `dual` setting, and even if we did that should be handled in `_params_from_cpu`, not `_attrs_from_cpu` (since it's a hyperparameter).

```suggestion
```